flink-core/src/main/java/org/apache/flink/configuration/CheckpointingOptions.java

```diff
@@ -169,6 +173,27 @@ public class CheckpointingOptions {
                                     + "deactivated. Local recovery currently only covers keyed state backends "
                                     + "(including both the EmbeddedRocksDBStateBackend and the HashMapStateBackend).");
 
+    /**
+     * This option configures local backup for the state backend, which indicates whether to make
+     * backup checkpoint on local disk. If not configured, fallback to {@link
+     * StateRecoveryOptions#LOCAL_RECOVERY}. By default, local backup is deactivated. Local backup
+     * currently only covers keyed state backends (including both the EmbeddedRocksDBStateBackend
+     * and the HashMapStateBackend).
+     */
+    public static final ConfigOption<Boolean> LOCAL_BACKUP_ENABLED =
```

Comment:
I suggest put this option at last of this class. Almost all of the options here will be deprecated and we can gather them on top of this class.